pkg/gql/query/types.go

```diff
@@ -53,7 +53,7 @@ var Header = graphql.NewObject(
 			"seed": &graphql.Field{
 				Type: Hex,
 			},
-			"txroot": &graphql.Field{
+			"generatorblspubkey": &graphql.Field{
```

Comment:
nitpick: we can also replace "txroot" with "generatorblspubkey" in graphql README as well.

In addition, in `wire.md` doc, we have:

| 32 | Tx Root | \[\]byte | Merkle root hash of all transactions in this block |